src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/PropertyPage.cs

```diff
@@ -363,7 +363,7 @@ public void SetObjects(uint cObjects, object[] ppunk)
                 }
             }
 
-            OnSetObjects(false);
+            WaitForAsync(() => OnSetObjects(false));
```

Comment:
Interestingly, this triggers a VSTHRD010 violation that doesn't fire without the switch to `/langver:8.0` and `<NullableReferenceTypes>true</NullableReferenceTypes>`.  Possible change in API behavior?

Reply:
❔ Was this intended to be fire-and-forget? Because this now changed the behavior to block on completion.